command/agent.go

```diff
@@ -28,16 +28,18 @@ import (
 	"github.com/hashicorp/vault/command/agent/auth/jwt"
 	"github.com/hashicorp/vault/command/agent/auth/kubernetes"
 	"github.com/hashicorp/vault/command/agent/cache"
-	"github.com/hashicorp/vault/command/agent/config"
+	agentConfig "github.com/hashicorp/vault/command/agent/config"
 	"github.com/hashicorp/vault/command/agent/sink"
 	"github.com/hashicorp/vault/command/agent/sink/file"
 	"github.com/hashicorp/vault/command/agent/sink/inmem"
 	gatedwriter "github.com/hashicorp/vault/helper/gated-writer"
 	"github.com/hashicorp/vault/sdk/helper/consts"
 	"github.com/hashicorp/vault/sdk/helper/logging"
+	"github.com/hashicorp/vault/sdk/logical"
 	"github.com/hashicorp/vault/sdk/version"
 	"github.com/kr/pretty"
 	"github.com/mitchellh/cli"
+	"github.com/pkg/errors"
```

Comment:
Did you intend to import this?

Reply:
Yes, its used here: https://github.com/hashicorp/vault/pull/7627/files#diff-cc3095270f1a410d27fb09bd8aade5ccR565

Reply:
There is a stdlib errors and usually it is unintended to pull in this package.

Reply:
Huh, yep you are right.  Not sure why my IDE brought in that one, but I fixed it.